addons/stock/static/src/views/search/stock_orderpoint_search_panel.js

```diff
@@ -1,16 +1,24 @@
 /** @odoo-module **/
 
-import { useState } from '@odoo/owl';
+import { useService } from "@web/core/utils/hooks";
+import { onWillStart, useState } from '@odoo/owl';
 import { SearchPanel } from "@web/search/search_panel/search_panel";
 
 
 export class StockOrderpointSearchPanel extends SearchPanel {
     static template = "stock.StockOrderpointSearchPanel";
 
     setup() {
+        this.orm = useService("orm");
         super.setup(...arguments);
         this.globalVisibilityDays = useState({value: 0});
         this.state.sidebarExpanded = false;
+        onWillStart(this.getVisibilityParameter);
+    }
+
+    async getVisibilityParameter() {
+        let res = await this.orm.call("ir.config_parameter", "get_param", ["stock.visibility_days", 0]);
+        this.globalVisibilityDays.value = Math.max(parseInt(res), 0);
```

Comment:
```suggestion
        this.globalVisibilityDays.value = Math.abs(parseInt(res)) || 0
```
I think this solution is better because if something else than a number is given to `Math.max`, it will return `NaN`, so if `parseInt` tries to parse something else than a number, it will return `NaN`.
And since `NaN` is a falsy value, `NaN || 0` will return 0.